interface/src/app/node/node.js

```diff
@@ -281,11 +282,13 @@ angular.module('adage.node', [
           $http.get(sampleURI, {params: {experiment: exp.accession}})
```

Comment:
We should discuss converting this to a service. Not for this PR, though, so I will flag this as a note in the issue I already created about restructuring the code.